maven-model-builder/src/main/java/org/apache/maven/model/building/DefaultModelBuilder.java

```diff
@@ -88,8 +91,10 @@
 import org.apache.maven.model.resolution.WorkspaceModelResolver;
 import org.apache.maven.model.superpom.SuperPomProvider;
 import org.apache.maven.model.validation.ModelValidator;
+import org.codehaus.plexus.interpolation.InterpolationException;
 import org.codehaus.plexus.interpolation.MapBasedValueSource;
 import org.codehaus.plexus.interpolation.StringSearchInterpolator;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
I'm still not comfortable with adding this class here. Others might feel differently.

Reply:
I added `static import` for `isNotEmpty` to show that I use only one method. 
But there is an unresolved discussion between you @slachiewicz and @rfscholte about it.

Reply:
@elharo I added my own `private static isNotEmpty` method. I want to avoid `lang3` vs `plexus` discussion because it's out of the scope of this PR. Can we proceed with this solution?